src/quantum/azext_quantum/operations/workspace.py

```diff
@@ -253,7 +254,7 @@ def create(cmd, resource_group_name=None, workspace_name=None, location=None, st
 
     deployment_async_operation = arm_client.deployments.begin_create_or_update(
         info.resource_group,
-        workspace_name,     # Note: This is actually specifying a the deployment name, but workspace_name is used here in test_quantum_workspace.py
+        DEPLOYMENT_NAME_PREFIX + workspace_name,
```

Comment:
is there a max size to the deployment name?

Reply:
Added code to truncate long deployment names to 64 characters.